packages/webcomponents/src/components/business-forms/payment-provisioning/payment-provisioning.css

```diff
@@ -2,15 +2,15 @@
   display: block;
 }
 
-*::part(label) {
+.body::part(label) {
```

Comment:
`part` selectors have lower specificity than css classes, so we are prepending the `.body` class to each `part` selector. The `.body` class is applied to any component using `StyledHost`